client/data/blogging-prompt/is-bloganuary.ts

```diff
@@ -1,9 +1,8 @@
 import { isEnabled } from '@automattic/calypso-config';
-
+import moment from 'moment';
 /**
- * In future this will be automatically enabled in January. For now it just checks a feature flag.
  * @returns true if bloganuary mode is active
  */
 export default function isBloganuary() {
-	return isEnabled( 'bloganuary' );
+	return moment( '2020-01-12' ).format( 'MM' ) === '01' || isEnabled( 'bloganuary' );
```

Comment:
I think this will always return true. Is this in for testing purposes?

Reply:
🤦 oops I didn't mean to push that! 😂